just-premium-htb.js

```diff
@@ -66,10 +66,170 @@ function JustPremiumHtb(configs) {
      */
     var __profile;
 
+    //? if (TEST) {
+    /**
+     * Necessary for tests, fake window object
+     * @type {{document: {createElement: fakeWindow.document.createElement, getElementsByTagName: fakeWindow.document.getElementsByTagName, cookie: string}}}
+     */
+    var fakeWindow = {
```

Comment:
Window object can be accessed via `Broswer. topWindow`. For a list of provided stub libraries, please refer to this file: 
https://github.com/indexexchange/just-premium-adapter/blob/development-v2/spec/support/libraryStubData.js